src/components/side_panel/pivot/pivot_layout_configurator/pivot_dimension/pivot_dimension.ts

```diff
@@ -27,6 +28,14 @@ export class PivotDimension extends Component<Props, SpreadsheetChildEnv> {
   static props = {
     dimension: Object,
     onRemoved: { type: Function, optional: true },
+    onNameUpdated: { type: Function, optional: true },
     slots: { type: Object, optional: true },
   };
+
+  private state: { isUpdate: boolean } = useState({ isUpdate: false });
+
+  updateName(ev: InputEvent) {
+    this.props.onNameUpdated?.(this.props.dimension, (ev.target as HTMLInputElement).value);
+    this.state.isUpdate = false;
```

Comment:
What do you mean by `isUpdate` ? Maybe `isUpToDate` ?

Reply:
I mean `I'm currently editing the title`. So no `isUpToDate`.
But i'm not convinced by `isUpdate` so if you have an idea...

Reply:
isUpdating ? Else I don't have any idea :/